modules/social_features/social_landing_page/social_landing_page.module

```diff
@@ -55,19 +55,7 @@ function social_landing_page_theme() {
   $theme_templates['node__featured'] = [
     'base hook' => 'node',
   ];
-  $theme_templates['node__event__featured'] = [
-    'base hook' => 'node',
-  ];
-  $theme_templates['node__topic__featured'] = [
-    'base hook' => 'node',
-  ];
-  $theme_templates['node__page__featured'] = [
-    'base hook' => 'node',
-  ];
-  $theme_templates['node__book__featured'] = [
-    'base hook' => 'node',
-  ];
-  $theme_templates['node__landing_page__featured'] = [
```

Comment:
Those definitions were needed only for the ability to have theme files used from a module, not a theme. 
Since we have common theme suggestions for all nodes - it will be still possible to use and override those and simal suggestions in themes.
So if you create a `node-something--featured.html.twig` file it will overwrite our `node--featured.html.twig` file for that bundle.